commit 7071bef27dae5c4d3426e1016398e500e9e24e2f
Author: Scott James Remnant <scott@canonical.com>
Date:   Tue Mar 3 14:24:52 2009 +0000

    UBUNTU: SAUCE: (no-up) hostap: Change initial operation mode to managed (infra)
    
    This was previously changed by using an "options" line in a modprobe.d
    file, however that practice is now deprecated.  This is because module
    names, option names, their values and even their current defaults can
    all change inside the kernel and module-init-tools has never been kept
    in sync.
    
    In addition, changing the kernel means that the option change will apply
    if the module is built in by users or the OEM team.
    
    Signed-off-by: Scott James Remnant <scott@canonical.com>
    Signed-off-by: Tim Gardner <tim.gardner@canonical.com>

diff --git a/drivers/net/wireless/hostap/hostap_hw.c b/drivers/net/wireless/hostap/hostap_hw.c
index 6df3ee5..90b7c9c 100644
--- a/drivers/net/wireless/hostap/hostap_hw.c
+++ b/drivers/net/wireless/hostap/hostap_hw.c
@@ -69,7 +69,7 @@ static char essid[33] = "test";
 module_param_string(essid, essid, sizeof(essid), 0444);
 MODULE_PARM_DESC(essid, "Host AP's ESSID");
 
-static int iw_mode[MAX_PARM_DEVICES] = { IW_MODE_MASTER, DEF_INTS };
+static int iw_mode[MAX_PARM_DEVICES] = { IW_MODE_INFRA, DEF_INTS };
 module_param_array(iw_mode, int, NULL, 0444);
 MODULE_PARM_DESC(iw_mode, "Initial operation mode");
 
